src/config/GeneralConfig.php

```diff
@@ -2795,6 +2795,23 @@ class GeneralConfig extends BaseConfig
      */
     public bool $storeUserIps = false;
 
+    /**
+     * @var string|null The handle of the Filesystem to use for storing temporary asset uploads. If not specified local temp folder will be used.
+     *
+     * ::: code
+     * ```php Static Config
+     * ->tempUploadsFs('myTempFs')
+     * ```
+     * ```shell Environment Override
+     * CRAFT_TEMP_UPLOADS_FS=myTempFs
+     * ```
+     * :::
+     *
+     * @group Assets
+     * @since 5.0.0
+     */
+    public ?string $tempUploadsFs = null;
```

Comment:
Should this be called `$tempAssetUploadsFs`, to clarify it is specifically for assets, and not all uploads? Or is that too verbose?

Reply:
Either one is fine with me. @brandonkelly - want to be a tiebreaker?

Reply:
Hmmm… there are several `*Upload*` config settings, none of which specify `asset`, but they also each could technically be respected by other upload functions if they existed.

Then there’s `craft\services\Path::getTempAssetUploadsPath()`, which does have `asset`.

So I guess for consistency with that, yes we should go with `tempAssetUploadFs`.

Reply:
`tempAssetUploadFs` it is!